autoload/neomake.vim

```diff
@@ -161,6 +159,33 @@ function! s:MakeJob(make_id, options) abort
             let maker = returned_maker
         endif
     endif
+    lockvar maker
+    let jobinfo.maker = maker
+
+    " Check for already running job for the same maker (from other runs).
+    " This used to use this key: maker.name.',ft='.maker.ft.',buf='.maker.bufnr
```

Comment:
When going back to using a key, this could be moved back again (to restart earlier).
Just using the maker name per buffer might be sane anyway?!